extensions/eda/rulebooks/demo_nautobot_rulebook.yml

```diff
@@ -0,0 +1,21 @@
+---
+# This example changes the polling interval to 5 seconds. It also adds query params to filter down the events coming into the platform.
+
+- name: "Watch for new changelog entries"
+  hosts: "localhost"
+  sources:
+    - jeffkala.nautobot_eda.nautobot_changelog:
```

Comment:
Can we migrate this to networktocode.nautobot or something of this nature?

Reply:
done, just missed changing it on my first pass.